api/all/src/main/java/io/opentelemetry/api/common/Value.java

```diff
@@ -84,6 +84,11 @@ static Value<List<KeyValue>> of(Map<String, Value<?>> value) {
     return KeyValueList.createFromMap(value);
   }
 
+  /** Returns a new {@link ValueBuilder} instance for creating an arbitrary {@link Value}. */
+  static ValueBuilder builder() {
```

Comment:
`ValueBuilder` may be a misnomer since you can only build "map" values using it

maybe?

```suggestion
  static MapValueBuilder mapBuilder() {
```

Reply:
went ahead and renamed, and introduced list builder as well